src/utils/test/patternlab-utils.js

```diff
@@ -27,24 +27,26 @@ async function getComponents() {
       directories.map((file, index) => {
         const builtComponentDir = PL_BUILD_PATTERNS_DIR + file;
         fs.readdir(builtComponentDir, (err, dir) => {
-          dir.forEach((component) => {
-            if (component.includes(".rendered.html")) {
-              const c = file + "/" + component;
-              components.push(c);
-            }
-            if (index === directories.length - 1) {
-              resolve(components);
+          // eslint-disable-next-line no-restricted-syntax
```

Comment:
what benefit or problem is this refactor solving?

Reply:
This file was reporting ESLint errors, this is just to get around that. Right now we're only using the compiled PL files to run our unit tests. They'll be removed moving forward.